flit-client/src/Conversation/Conversation.css

```diff
@@ -0,0 +1,8 @@
+.conversation {
+    border: 1px solid black;
+}
+
+.header {
+    font-size: 20px;
+    font-weight: 800;
```

Comment:
As a note, we need to explicitly import any weights we want to use as part of the import from Google Fonts, or else they will be faked - badly - by the browser or snapped to the nearest size we did import.